rook/Tester.py

```diff
@@ -469,6 +570,22 @@ def run_backend(self, _):
         self.results.message += "\n" + message
     return self.results
 
+  def _wait_for_all_written(self):
+    """
+      Waits until all the files for the differ have been written
```

Comment:
No In/Out docstrings included, and I don't think the standards have an exception for argument- and return-less methods.

Reply:
Fixed.